_includes/releases/v22.1/v22.1.0-beta.5.md

```diff
@@ -0,0 +1,52 @@
+## v22.1.0-beta.5
+
+Release Date: May 3, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-beta-5-security-updates">Security updates</h3>
+
+- `crdb_internal.reset_sql_stats()` and `crdb_internal.reset_index_usage_stats()` [built-in functions](../v22.1/functions-and-operators.html#system-info-functions) now check if user has the [admin role](../v22.1/security-reference/authorization.html#admin-role). [#80384][#80384]
```

Comment:
Just a small typo here, missed "the" before user:

"... now check if the user has"